qtpy/cli.py

```diff
@@ -0,0 +1,81 @@
+import argparse
+import inspect
+
+
+class RawDescriptionArgumentDefaultsHelpFormatter(
+    argparse.RawDescriptionHelpFormatter,
+    argparse.ArgumentDefaultsHelpFormatter,
+):
+    pass
+
+
+def cli():
```

Comment:
```suggestion
def cli(cli_args=None):
```

Wouldn't hurt to have this, in case we want to pass in arguments programmatically